jwt/jwt.go

```diff
@@ -70,7 +70,12 @@ func NewParserWithOpts(keysProvider KeysProvider, logger log.FieldLogger, opts P
 		audienceMatchers = append(audienceMatchers, glob.Compile(audPattern))
 	}
 	return &Parser{
-		parser:                        jwtgo.NewParser(jwtgo.WithExpirationRequired()),
+		parser: func() *jwtgo.Parser {
```

Comment:
Suggest defining `[]jwtgo.ParserOption` slice and using it in the `jwtgo.NewParser()` constructor.